deploy.sh

```diff
@@ -185,6 +185,7 @@ for proj in $projects; do # bash only
   # but that will not work when deploying to staging or production.
   if command -v gcloud &> /dev/null ; then
     export SENTRY_AUTH_TOKEN=$(gcloud secrets versions access latest --secret="SENTRY_AUTH_TOKEN")
+    #export CODECOV_TOKEN2=$(gcloud secrets versions access latest --secret="CODECOV_TOKEN")
```

Comment:
iirc this is only needed for ASP.NET, where debug file upload happens inside GCP instead of GitHub Actions.